src/app/utils/storage.ts

```diff
@@ -30,7 +30,7 @@ class ChromeStorage {
     });
   }
 
-  getItem(key: string) {
+  getItem(key: string): Promise<any> {
```

Comment:
Might be nice to make this generic but default to any:
```suggestion
  getItem<T = any>(key: string): Promise<T> {
```

Reply:
nice